### DISCUSSION TITLE
文章顶部栏

### DISCUSSION THREAD

**Discussion by yefengwu** [liuzhihang/halo-theme-hao#35] [Ideas]
能否提供一个关闭文章顶部栏的开关，现在加载的颜色太亮了，感觉不舒服。
![image](https://user-images.githubusercontent.com/45461486/207296675-0ad2b599-07f0-408e-8c87-97b01f87d4fb.png)

**Comment by liuzhihang** [comment]
哪里很亮，是把图片这里关了么？ 还是导航栏关了

**Comment by yefengwu** [comment]
是这个，这个可以自己调颜色吗？
![GIF 2022-12-14 9-46-09](https://user-images.githubusercontent.com/45461486/207484977-d9f46588-5477-4988-afcf-f29bfabb088d.gif)

**Reply by yefengwu** [reply]
文章进入时顶部标题栏的渐变色

**Reply by sunheyi6** [reply]
不行，默认是根据图片来自动识别主要颜色的

**Comment by lianlianlianlianlianlian** [comment]
我觉得很好 不用改
